packages/fx-core/src/plugins/resource/aad/aadAppClient.ts

```diff
@@ -218,6 +229,7 @@ export class AadAppClient {
         if (retries === 0) {
           throw error;
         } else {
+          TelemetryUtils.sendEvent(stage, {[Telemetry.methodName]: fn.toString(), [Telemetry.retryTimes]: (10-retries).toString()});
           await delay(5000);
```

Comment:
`we will send telemetry every time it is retrying`. Just want to confirm whether the behavior is expected?
If we retry 7 times, then we will have 7 telemetry event.

Reply:
Maybe we can only send 1 telemetry?

Reply:
Actually adding this telemetry is for get how many times some user will wait. In some cases user will just close the session and the error telemetry will not be sent.

Reply:
Just a reminder. If we send multiple events, maybe we also should consider how to utilize it and show it in Power BI.

Reply:
Since we sent multiple error telemetries for one operation, the error rate will be **higher** than actual value (not sure whether we can workaround this issue via Kusto query). Can we confirm whether it's fine?
Another option is send a new event for the errors and exclude this new event in our report to avoid possible confusion on error report..

Reply:
Here I send a telemetry event without success = no, thus I think it seems will not cause higher error rate. To get detailed info, we can do like this: count(retry = 1) = telemetry(retry = 1) - telemetry(retry=2). Where count() stands for the actual number of user retry 1 time, and telemetry() means the number of telemetry with specific retry times.